### PR TITLE
testing/tcolors: new aport

### PR DESCRIPTION
**Pull request by jjsearle**

### CODE REVIEW

File: testing/tcolors/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Contributor: Joe Searle <joe@jsearle.net>
+# Maintainer: Joe Searle <joe@jsearle.net>
+pkgname=tcolors
+pkgver=0.3.1
+pkgrel=0
+pkgdesc="Commandline color picker and palette builder"
+url="https://github.com/bcicen/tcolors"
+arch="all"
+license="MIT"
+makedepends="go gzip"
```
Comment (Cogitri) [inline testing/tcolors/APKBUILD]: busybox gzip doesn't do the trick?

File: testing/tcolors/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/bcicen/tcolors/archive/v$pkgver.tar.gz"
+options="!check" # No tests
+
+build() {
+	gzip -f -k docs/man1/tcolors.1
```
Comment (Cogitri) [inline testing/tcolors/APKBUILD]: Why do you manually compress the manpage here? Abuild should do that do you in $pkgname-doc's default function.

Comment (jjsearle) [reply testing/tcolors/APKBUILD]: I wasn't aware it did this. I have removed the gzip and retested and its fine, thanks.

File: testing/tcolors/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+build() {
+	go mod download
+	go build -ldflags "-w -X main.version=${pkgver}-${pkgrel} -X main.build=$pkgbuild" -o tcolors
+}
+
+package() {
+	install -Dm755 tcolors "${pkgdir}/usr/bin/tcolors"
+	install -Dm644 docs/man1/tcolors.1 "$pkgdir/usr/share/man/man1/${pkgname}.1"
+	install -Dm644 LICENSE -t "$pkgdir/usr/share/licenses/$pkgname"
```
Comment (Cogitri) [inline testing/tcolors/APKBUILD]: Ah, no need to install licenses for SPDX licenses (so all licenses listed on https://spdx.org/licenses)

Comment (jjsearle) [reply testing/tcolors/APKBUILD]: corrected :+1:

**Pull request closed without merging**